app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -295,11 +295,20 @@ <h2>Upload Sessions</h2>
                  aria-labelledby="v-pills-questions-tab">
                 <h2>Questions</h2>
 
-                <ul class="list-group list-group-flush">
+                <script src="https://cdn.jsdelivr.net/npm/sortablejs@latest/Sortable.min.js"></script>
+
+                <ul class="list-group list-group-flush" id="full-questions-list">
                     {% for question in object.questions.all %}
-                        <li class="list-group-item">
+                        <li class="list-group-item" gc-update-link="{% url 'api:reader-studies-question-detail' pk=question.pk %}">
```

Comment:
Should I use the REST API like this? Or should I rather submit to a local API endpoint in the view?

Reply:
No, use django forms.

Reply:
In general, don't use custom attributes, if you need to pass extra things use data attributes: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes